audio_service/lib/audio_service.dart

```diff
@@ -3243,15 +3293,15 @@ class _HandlerCallbacks extends AudioHandlerCallbacks {
         ));
       });
     }
-    return await handler.getChildren(parentMediaId, options);
+    return handler.getChildren(parentMediaId, options);
```

Comment:
On this occasion I'm fine either way, but note that if one of the arguments needed to be awaited, I'd just rather use await everywhere. This would be a good example of something I'd not want tied down in any coding standard and probably a policy of not changing it for smaller diffs.